packages/ui/src/Select/SelectField.tsx

```diff
@@ -56,7 +56,10 @@ export type SelectFieldExtraProps<V> = {
   singleValueTooltipVisible?: boolean
 } & DataTestProp &
   Pick<InputHTMLAttributes<HTMLElement>, 'autoFocus' | 'disabled' | 'required' | 'placeholder'> &
-  Pick<ReactSelectProps, 'isSearchable' | 'isClearable' | 'menuIsOpen' | 'menuPlacement' | 'noOptionsMessage' | 'inputValue'> &
+  Pick<
+    ReactSelectProps,
+    'isSearchable' | 'isClearable' | 'menuIsOpen' | 'menuPlacement' | 'closeMenuOnScroll' | 'noOptionsMessage' | 'inputValue'
+  > &
```

Comment:
Please enable closeMenuOnScroll when menuPortalTarget = 'body'

Reply:
It's experimental.
I have a plan to do this later, after checking how it works in our application.
This option doesn't seem to work properly with complex pages and iframe (storybook). Maybe we need a new code to do this. 
And changes in loki's screenshots are very small and minor